test/integration/fixtures/options/require/root-hook-defs-b.fixture.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+exports.mochaHooks = {
+  beforeAll: [
+    function() {
+      console.log('beforeAll array 1');
+    },
+    function() {
+      console.log('beforeAll array 2');
+    }
+  ],
+  beforeEach: [
+    function() {
+      console.log('beforeEach array 1');
+    },
+    function() {
+      console.log('beforeEach array 2');
+    }
+  ],
+  afterAll: [
+    function() {
+      console.log('afterAll array 1');
+    },
+    function() {
+      console.log('afterAll array 2');
+    }
+  ],
+  afterEach: [
+    function() {
+      console.log('afterEach array 1');
+    },
+    function() {
+      console.log('afterEach array 2');
```

Comment:
Do you have tests for async functions as hooks?

Reply:
not in the integration tests, but in the [unit tests](https://github.com/mochajs/mocha/pull/4244/files#diff-a8ee42e10c11a0f7455ec42d9c22503aR28)